tests/network/test_character_control.py

```diff
@@ -99,3 +100,29 @@ def test_bob_character_control_retrieve(bob_control, enacted_federated_policy, c
 
     del(request_data['alice_signing_pubkey'])
     response = bob_control.put('/retrieve', data=json.dumps(request_data))
+
+
+def test_enrico_character_control_encrypt_message(enrico_control):
+
+    request_data = {
+        'message': b64encode(b"The admiration I had for your work has completely evaporated!").decode(),
+    }
+
+    response = enrico_control.post('/encrypt_message', data=json.dumps(request_data))
+    assert response.status_code == 200
+
+    response_data = json.loads(response.data)
+    assert 'message_kit' in response_data['result']
+    assert 'signature' in response_data['result']
+
+    # Check that it serializes correctly.
+    message_kit = UmbralMessageKit.from_bytes(
+                            b64decode(response_data['result']['message_kit']))
+
+    # Send bad data to assert error return
+    response = enrico_control.post('/encrypt_message', data='bad')
+    assert response.status_code == 400
```

Comment:
Not immediately evident to me why this is a 400 and not a 5xx - is this the problem you were telling me about today?